_includes/social_links/dribbble.html

```diff
@@ -0,0 +1 @@
+<a href="https://dribbble.com/{{ include.username || site.dribbble_username }}"><svg class="svg-icon" xmlns="http://www.w3.org/2000/svg"><use xlink:href="/assets/icons.svg#dribbble"></use></svg> <span class="username">{{ site.dribbble_username }}</span></a>
```

Comment:
This was changed in Jekyll 3.5 AFAIR.. so we should be fine since we're moving to depend on at least v3.5 from the next release onwards